.planning/phases/11-self-assessment-persistence-topics/11-03-PLAN.md

```
@@ -56,242 +57,304 @@ Output: 3 standalone HTML explorer files in dist/, verified working by both auto
+# Topic data integrity
+echo "Topic data integrity:"
+for explorer in dividend-strategy options-greeks risk-management; do
+  bun -e "
+    const html = await Bun.file('./src/explorer/dist/${explorer}-explorer.html').text();
+    const match = html.match(/const TOPIC_DATA = ({.*?});/s);
+    if (!match) { console.error('  ${explorer}: NO TOPIC_DATA'); process.exit(1); }
+    const d = JSON.parse(match[1]);
+    console.log('  ${explorer}: ' + d.nodes.length + ' nodes, ' + d.edges.length + ' edges, slug=' + d.metadata.slug);
+  "
+done
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)